docs/source/codependence/correlation_based_metrics.rst

```diff
@@ -168,12 +219,18 @@ The following examples show how the described above correlation-based metrics ca
 .. code-block::
 
     import pandas as pd
+    import numpy as np
     from mlfinlab.codependence import distance_correlation, angular_distance,
```

Comment:
brackets around the imports - since it runs over multiple lines.

Reply:
Needed to resolve conflicts and fixed that.